rbac/common/sawtooth/__init__.py

```diff
@@ -12,3 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # -----------------------------------------------------------------------------
+
+from rbac.common.sawtooth.client_sync import ClientSync
+
+# pylint: disable=invalid-name
+client = ClientSync()
+
+__all__ = ["client"]
```

Comment:
Why is this necessary? Are we doing a '*' import on client somewhere?

Reply:
marks client as a variable for purpose of export, not an unused variable

Reply:
It specifies what will be pulled in when a 'from sawtooth import *' is performed, right?  So it filters out any additional variables that would otherwise be pulled in on a wildcard import.

Reply:
It marked the variable for purposes of export. It may not be required here, but it is good practice. It is required when you import a variable that you intend to export, as the linter will suggest it is an unused variable, and the app will break when you remove it.